app/data_models/app_models.py

```diff
@@ -34,19 +41,31 @@ def __init__(self, eq_session_id, user_id, expires_at, session_data):
 
 
 class UsedJtiClaim:
-    def __init__(self, jti_claim, expires_at):
+    def __init__(self, jti_claim: str, expires_at: datetime) -> None:
         self.jti_claim = jti_claim
         self.expires_at = expires_at.replace(tzinfo=timezone.utc)
 
 
 # pylint: disable=no-self-use
 class Timestamp(fields.Field):
-    def _serialize(self, value, *args, **kwargs):  # pylint: disable=unused-argument
+    # pylint: disable=unused-argument
+    def _serialize(
+        self,
+        value: datetime,
+        *args: Optional[list],
+        **kwargs: Any,
```

Comment:
This causes mypy error probably because parent class (`Field`) in fields.py doesn't have type declared for **kwargs:
`def _serialize(self, value: typing.Any, attr: str, obj: typing.Any, **kwargs):`. Can't really change it cause it calls standard library class.